### PR TITLE
Fix F108 title, tweak G219

### LINKED ISSUE #3128: G219 for 2.5.7 Dragging Movement is misleading
The title of G219 currently reads: "Ensuring that a single pointer alternative is available for dragging movements that operate on content"

This is slightly misleading, because it seems to suggest that dragging movements *aren't* single pointer, when in fact they are/can be.

Suggest rewording to be more specific/clear: "Ensuring that a non-dragging-based single pointer alternative is available for dragging movements that operate on content"

EDIT: same issue is present in the actual body of the technique, making a false dychotomy between "single pointer" and "dragging movements".

https://www.w3.org/WAI/WCAG22/Techniques/general/G219

### PR DESCRIPTION
**Pull request by patrickhlauke**
* for G219, the title and wording are incorrect. they seem to suggest that "dragging movements" are not "single pointer" gestures, which they are (if they only involve a single pointer). what counts is that the alternative must be *non-dragging-based* and single pointer. Closes #3128 
* for F108, the title is incomplete (from when the SC number, and final title of the SC, weren't finalised yet)

### CODE REVIEW

**Comment by mbgower** [comment]
I'm not entirely convinced that the title needs to be altered, but I do think the wording changes in the techniques is good.
To me, if the title was simply "Ensuring that an alternative is available for [dragging movement](https://www.w3.org/TR/WCAG22/#dfn-dragging-movement)s that operate on content" no one would likely be quibbling about it, since obviously the need is an alternative to dragging. Saying non-dragging here seems pretty redunandant, at least in the title.
So, one way to shorten this title and make it clearer could be to actually just remove 'single-pointer'. All the changes in the document could still stand -- because of course it needs to be a single-pointer action to meet 2.1.

File: techniques/general/G219.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 	<head>
-		<title>Ensuring that a single pointer alternative is available for dragging movements that operate on content</title>
+		<title>Ensuring that a non-dragging-based single pointer alternative is available for dragging movements that operate on content</title>
```
Comment (mbgower) [inline techniques/general/G219.html:4]:
```suggestion
		<title>Ensuring that an alternative is available for dragging movements that operate on content</title>
```

File: techniques/general/G219.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
 	</head>
 	<body>
-    <h1>Ensuring that a <a>single pointer</a> alternative is available for <a>dragging movement</a>s that operate on content</h1>
+    <h1>Ensuring that a non-dragging-based <a>single pointer</a> alternative is available for <a>dragging movement</a>s that operate on content</h1>
```
Comment (mbgower) [inline techniques/general/G219.html:8]:
```suggestion
    <h1>Ensuring that an alternative is available for <a>dragging movement</a>s that operate on content</h1>
```

**Review by mbgower — COMMENTED** [review]
As I mentioned in the conversation, all changes make sense. I've made a suggestion for shortening the general technique.

**Comment by patrickhlauke** [comment]
sure, not fussy about either approach, as long as the currently misleading one is changedOn 3 Apr 2023, at 17:03, Mike Gower ***@***.***> wrote:﻿
I'm not entirely convinced that the title needs to be altered, but I do think the wording changes in the techniques is good.
To me, if the title was simply "Ensuring that an alternative is available for dragging movements that operate on content" no one would likely be quibbling about it, since obviously the need is an alternative to dragging. Saying non-dragging here seems pretty redunandant, at least in the title.
So, one way to shorten this title and make it clearer could be to actually just remove 'single-pointer'. All the changes in the document could still stand -- because of course it needs to be a single-pointer action to meet 2.1.

**Comment by patrickhlauke** [comment]
currently on the road, let me check this when i'm back thursday night before doing anything with it...

**Comment by patrickhlauke** [comment]
updated the link *to* the technique with the shorter title

**Review by detlevhfischer — APPROVED** [review]

**Comment by detlevhfischer** [comment]
Looks like a no-brainer. Ready for survey?

File: techniques/general/G219.html [hunk 3]
```diff
@@ -18,9 +18,9 @@ <h2>Applicability</h2>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this Technique is to ensure that people with motor impairments who cannot carry out <a>dragging movements</a> are presented with an alternative <a>single pointer</a> interaction.</p> 
+			<p>The objective of this Technique is to ensure that people with motor impairments who cannot carry out <a>dragging movements</a> are presented with an alternative <a>single pointer</a> interaction that does not involve dragging.</p>
```
Comment (mbgower) [inline techniques/general/G219.html:21]: Was this single pointer mention intentionally preserved, or was it an oversight?

File: techniques/general/G219.html [hunk 4]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 	<head>
-		<title>Ensuring that a single pointer alternative is available for dragging movements that operate on content</title>
+		<title>Ensuring that an alternative is available for dragging movements that operate on content</title>
```
Comment (alastc) [inline techniques/general/G219.html:4]:
```suggestion
		<title>Ensuring an alternative to dragging movements</title>
```

File: techniques/failures/F108.html [hunk 5]
```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 <head>
-      <title>Failure of Success Criterion 2.5.X Dragging due to not providing a single pointer method for the user to operate a function that does not require a dragging movement</title>
+      <title>Failure of Success Criterion 2.5.7 Dragging Movements due to not providing a single pointer method for the user to operate a function that does not require a dragging movement</title>
```
Comment (alastc) [inline techniques/failures/F108.html:4]:
```suggestion
      <title>Failure of Success Criterion 2.5.7 due to not providing a single pointer method for the user to operate a function that does not require a dragging movement</title>
```
